tests/vision/checks/performance/robustness_report_test.py

```diff
@@ -48,6 +49,49 @@ def test_mnist(mnist_dataset_train, mock_trained_mnist, device):
     }))
 
 
+def test_mnist_torch(mnist_dataset_train_torch, mock_trained_mnist, device):
```

Comment:
We should also have a test for some RGB dataset.
For all I care, can create a fixture that has an version of MNIST that is RGB instead of greyscale.

Reply:
We can just test on COCO